config.env.py

```diff
@@ -14,6 +14,7 @@
 
 # Logging config
 LOG_LEVEL = environ.get("PACKET_LOG_LEVEL", "INFO")
+ANALYTICS_ID = environ.get("ANALYTICS_ID", "UA-134137724-9")
```

Comment:
You might want to have the default be a dummy value instead of a real ID. If this is the prod one for packet it could cause actions on local dev sessions to get attributed to the prod instance.